CallAutomation_SimpleIvr/appsettings.json

```diff
@@ -9,6 +9,7 @@
   "ACSAlternatePhoneNumber": "%ACSAlternatePhoneNumber%", // get it from acs resouce
   "ParticipantToAdd": "%ParticipantToAdd%",
   "BaseUri": "%BaseUri%",
+  "declinecall": true,
```

Comment:
May be we required both decline and accept call for same deployment. 
use declineCallForUsers= "comma separated phone, useridentifier"
if the callerId belong from this group decline.